crowdloan/polkadot.json

```diff
@@ -218,5 +218,13 @@
     "description": "Bifrost is a DeFi protocol that connect with Polkadot and different PoS blockchains to provide staking derivative though XCMP or Smart Contract",
     "website": "https://bifrost.finance/",
     "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/white/Bifrost.svg"
+  },
+  {
+    "paraid": "2043",
+    "name": "OriginTrail",
+    "token": "OTP",
+    "description": "OriginTrail - The world’s first Decentralized Knowledge Graph️",
```

Comment:
OriginTrail is the world's first Decentralized Knowledge Graph️

Reply:
fixed